mesonbuild/interpreter.py

```diff
@@ -216,6 +217,20 @@ def get_method(self, args, kwargs):
             return args[1]
         raise InterpreterException('Entry %s not in configuration data.' % name)
 
+    def get_unquoted_method(self, args, kwargs):
+        if len(args) < 1 or len(args) > 2:
+            raise InterpreterException('Get method takes one or two arguments.')
+        name = args[0]
+        if name in self.held_object:
+            val = self.held_object.get(name)[0]
+        elif len(args) > 1:
+            val = args[1]
+        else:
+            raise InterpreterException('Entry %s not in configuration data.' % name)
+        if val[0] == '"' and val[-1] == '"':
+          return val[1:len(val)-1]
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four
* [E226] missing whitespace around arithmetic operator

<!-- Comment by [SideCI](https://sideci.com) -->